library/Fission/Web/DNS.hs

```diff
@@ -18,7 +19,10 @@ type API = Capture "cid" CID
         :> PutAccepted '[PlainText, OctetStream] URL.DomainName
 
 server :: MonadDNSLink m => Entity User -> ServerT API m
-server (Entity _id User { userUsername }) cid =
+server = updateLink
+
+updateLink :: MonadDNSLink m => Entity User -> CID -> m URL.DomainName
+updateLink (Entity _id User { userUsername }) cid =
```

Comment:
Why?

Reply:
I assumed we wanted the specs to be independent of Servant, hence this code. Didn't go through with the specs after all the refactoring.

Reply:
I see; gotcha.

`ServerT api` is just a fancy type alias. No need to break them out into separate functions 👍

Reply:
Just put it back as before.